src/components/Description/index.jsx

```diff
@@ -0,0 +1,14 @@
+
+export default function Description(){
+    return (<div className={"place-items-center m-auto"}>
+                <div className={"container m-auto px-[5rem]"}>
+                    <div className={"mx-[5rem] min-w-full"}>
+                        <h1>Description</h1>
+                    </div>
+                    <div className={"ml-[10rem] min-w-fit"}>
+                        <body>Flourish! is an annually held conference intendedc to introduce both CS and non-CS, technical and non-technical majors to the world of open source through speakers, demos, workshops, sponsors, job opportunities, and other events! We show off hardware, software, business, entrepreneurship, finance, and more aspects of open source. We want to make this project bigger and bigger year over year and bring in new students. Our previous conferences focused on a small number of students in a technical software direction, but we are looking to change that!</body>
```

Comment:
Didn't realize the who we are section used body tags, but that is incorrect and can cause problems down the road. Could you please change this to something else? (p would be best)